src/main/resources/org/jenkinsci/plugins/casc/ObsoleteConfigurationFormat/message.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <div class="error">
```

Comment:
IIUC wrong classes are used here. Administrative monitors have been reworked: https://jenkins.io/blog/2018/01/21/overhaul-of-manage-jenkins-page/#for-plugin-developers . CC @recena

Reply:
The blog post does a poor job on the example.